app/scripts/modules/ecs/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -97,6 +97,15 @@ export interface IEcsTaskDefinitionArtifact {
   artifactId?: string;
 }
 
+export interface IEcsServiceDiscoveryRegistry {
```

Comment:
since this isn't used in this file, why not define it in `ServiceDiscovery.tsx` ?

Reply:
Good catch! Initially thought i might need it here, but didnt end up using it. Moved it.